huaweicloud/resource_huaweicloud_cce_node_v3_test.go

```diff
@@ -137,6 +143,24 @@ func testAccCheckCCENodeV3Exists(n string, cluster string, node *nodes.Nodes) re
 	}
 }
 
+func testAccCCENodeImportStateIdFunc() resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		var clusterID string
+		var nodeID string
+		for _, rs := range s.RootModule().Resources {
```

Comment:
suggest to use `s.RootModule().Resources["huaweicloud_cce_cluster.test"]` and `s.RootModule().Resources["huaweicloud_cce_node.test"]`, you can refer to `testAccCheckCCENodeV3Exists` function.